kore/app/exec/Main.hs

```diff
@@ -395,6 +400,8 @@ mainWithOptions
                           where
                             failure pat = (ExitFailure 1, pat)
                             success = (ExitSuccess, mkTop $ mkSortVariable "R")
+                            unknown = (ExitSuccess,
+                                       mkTop $ mkSortVariable "Unknown")
```

Comment:
According to the style guide, indentation should use a number of spaces divisible by 4 except in certain cases where it makes sense to use half-indentation (e.g. for the `where` keyword). I think that this line uses 4n+3 spaces.

Reply:
done